.storybook/.babelrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
         },
       },
     ],
+    "@babel/preset-react"
```

Comment:
Why do we need this within this PR?

Reply:
Because of the new changelog addon. The SB addonapi just renders react components. And without that there was an error.